android/app/src/main/java/com/goober/MainApplication.java

```diff
@@ -25,6 +26,7 @@ protected boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new RCTCameraPackage(),
```

Comment:
I think it attached automatically when the library has installed. you can resolve it.